### ISSUE TITLE
[GaugeTile] accessibility violations in storybook

### ISSUE THREAD

**Issue by tay1orjones** [carbon-design-system/carbon-addons-iot-react#1438]
### Describe the bug

From [Internal Watson-IoT/pal-tracking#354](https://github.ibm.com/Watson-IoT/pal-tracking/issues/354) from @JordanWSmith15 

> Three violations in the Graphite storybook from the gauge tile
> <img width="1678" alt="Screen Shot 2020-04-01 at 7 02 57 PM" src="https://media.github.ibm.com/user/17018/files/54163e00-7587-11ea-9837-8841fbdca4de">
> https://pages.github.ibm.com/maximo-app-framework/graphite/react/storybook/?path=/story/components-%E2%98%A2-dashboard--value-gauge
> 
> In the IoT repo, you can see 6 violations in the basic dashboard tile (only place where the gauge tile is demo'ed), but it is more difficult to pinpoint them there: https://carbon-addons-iot-react.com/?path=/story/watson-iot-dashboard--basic-dashboard
